pointers/pointers.c

```diff
@@ -66,7 +92,23 @@ int string_compare(char *m, char *n)
 */
 char *find_string(char *haystack, char *needle)
 {
-
+    while (*haystack)
+    {
+        char *begin = haystack;
```

Comment:
the way your using your pointers looks good Eva!, let me know if you have any other questions regarding pointers :)